docs/dev_coding_standard.rst

```diff
@@ -10,6 +10,20 @@ In essence we seek code that looks good, is easy to read and has great documenta
 The following is a list of exceptions/deviations from the google style guide that have been agreed upon for libCellML software development purposes.  Think of it like case-law.
 
  * lower camel case class method names
+ * indent code 4 spaces at a time (no tabs)
+
+Doxygen Comments
+================
+
+ * Doxygen comments should be sentence-style: beginning with capitalisation and ending with punctuation. However, they do not need to form grammatically correct sentences.
+
+   * An exception for capitalisation is code-words (e.g., true, false, std::string), which should be styled as typewriter text with a preceding "@c".
```

Comment:
Actually, Doxygen picks up class names as long as they are capitalised and links back to the relevant class in the generated documentation. For instance, [here's the generated version](http://autotest.bioeng.auckland.ac.nz/libcellml-buildbot/b8a163fb017ccc9def0bd1eea60b7f25f9020705/dox/code/classlibcellml_1_1NamedEntity.html#a21ef5ac512cea802de725a0a729cf970) of the Entity class mention in the getName function you referred to below.